render-test-config.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# TODO: Add Janitor secrets here
+# https://broadworkbench.atlassian.net/browse/ID-96
```

Comment:
I fleshed out the https://broadworkbench.atlassian.net/browse/ID-96 description a bit more